app/gui/view/documentation/src/lib.rs

```diff
@@ -83,9 +85,19 @@ pub const VIEW_WIDTH: f32 = 300.0;
 /// Height of Documentation panel.
 pub const VIEW_HEIGHT: f32 = 300.0;
 
+/// "Hovered item preview" caption height in pixels.
+const CAPTION_HEIGHT: f32 = 30.0;
+/// Caption animation is faster than standard animation.
+const CAPTION_ANIMATION_SPRING_MULTIPLIER: f32 = 1.5;
+/// The caption is hidden if its height is less than this value.
+const MIN_CAPTION_HEIGHT: f32 = 1.0;
```

Comment:
Can we put those to styles?

Reply:
I moved all constants (except for a few) to the stylesheet and refactored the implementation.